awx/main/tasks.py

```diff
@@ -354,6 +355,21 @@ def send_notifications(notification_list, job_id=None):
 
 @task(queue=get_local_queuename)
 def gather_analytics():
+    def _gather_and_ship(subset, since=None, until=None):
+        error = 0
+        try:
+            tgz = analytics.gather(subset=subset, since=since, until=until)
+            if not tgz:
+                return error
+            logger.info('gathered analytics: {}'.format(tgz))
+            analytics.ship(tgz)
+        except Exception:
+            error += 1
```

Comment:
let log the exception, so it doesn't get lost:

```
except Exception as e:
    logging.error(e, exc_info=True)
```

Reply:
Updated with logger.exception